packages/checkbox/src/theme.ts

```diff
@@ -0,0 +1,45 @@
+import { colors } from '@xcritical/theme';
+
+import { CheckboxTheme } from './interfaces';
+
+
+export const checkboxThemeNamespace = '@xcritical\\checkbox';
+
+export const defaultCheckboxTheme: CheckboxTheme = {
+  appearance: {
```

Comment:
You don't need appearance -> default.
Just write theme styles (it is already default - **default**CheckboxTheme).